mergify_engine/tests/unit/test_subscription.py

```diff
@@ -155,7 +155,7 @@ async def test_active_feature(redis_cache):
         "friend",
         frozenset([subscription.Features.PRIORITY_QUEUES]),
     )
-    assert sub.has_feature(subscription.Features.PRIORITY_QUEUES) is False
+    assert sub.has_feature(subscription.Features.PRIORITY_QUEUES) is True
```

Comment:
Not sure I get why this one change.

Reply:
The value was obviously wrong before.

Reply:
It changed because we don't check `self.active` in `has_feature` anymore.
I agree the test before was a bit wrong but it was the logic we implemented. I think it's better now.

Reply:
glad we drop `self.active` then :)